pkgs/by-name/gi/git-blame-someone-else/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation {
+  pname = "git-blame-someone-else";
+  version = "0-unstable-6-13-2018";
+
+  src = fetchFromGitHub {
+    owner = "jayphelps";
+    repo = "git-blame-someone-else";
+    rev = "8d854c2d78cb98afdb9f5a73240e06393260b327";
+    hash = "sha256-xraG1dR5Q8oDlUXARgh0ql8eRwH4bJWblJFjH1wJcys=";
+  };
```

```suggestion
  };

  makeFlags = [ "prefix=$(out)" ];
```
Doesn't build otherwise.